File: Maximum-Number-of-Words-Found-in-Sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int num = 0, maxNum = 0;
        for (auto& s : sentences) {
            num = count(s.begin(), s.end(), ' ') + 1;
            maxNum = max(num, maxNum);
        }
        return maxNum;
    }
};

File: Maximum-Number-of-Words-Found-in-Sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function (sentences) {
    let num = 0, maxNum = 0;
    for (const s of sentences) {
        num = s.split(' ').length;
        maxNum = Math.max(num, maxNum);
    }
    return maxNum;
};